ferrocene/doc/evaluation-report/src/rustfmt/use-cases.rst

```diff
@@ -0,0 +1,75 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+.. default-domain:: qualification
+
+Use Cases
+=========
+
+Installing rustfmt
+------------------
+
+.. id:: RUSTFMT_UC0_INST
+
+**Actor(s):** User, tar (compression utility)
+
+**Input:** A rustfmt release tarball.
+
+**Output:** The rustfmt binary in the Ferrocene directory.
+
+**Environment constraints:** Tar is correctly installed.
```

Comment:
```suggestion
**Environment constraints:** tar is correctly installed.
```

Reply:
Also changed it in rustc, to stay consistent